zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -137,6 +156,23 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
         );
 
         // Transfer value from caller to callee
+        let intrinsic_tx_value = cb.query_word();
+        cb.condition(is_tx_invalid.expr(), |cb| {
+            cb.require_equal(
+                "intrinsic_tx_value == 0",
+                intrinsic_tx_value.expr(),
```

Comment:
The real transfer value should be from_bytes::expr(& intrinsic_tx_value.cells). 
intrinsic_tx_value.expr() is the RLC value, although they are the same in this 0 case.

Reply:
Updated. Why we need to use `from_bytes::expr(& intrinsic_tx_value.cells)`?

Reply:
Could we change `from_bytes::expr(& intrinsic_tx_value.cells)` to `rlc::expr(&[intrinsic_tx_value.expr()], cb.power_of_randomness())`?

Reply:
Change to using `IsZeroGadget`.

Reply:
Change to using BatchedIsZeroGadget.